packages/pwa-buildpack/lib/Utilities/__tests__/createDotEnvFile.spec.js

```diff
@@ -9,7 +9,7 @@ const createDotEnvFile = require('../createDotEnvFile');
 const prettyLogger = require('../../util/pretty-logger');
 
 const snapshotEnvFile = contents =>
-    contents.replace(/.+?Generated by @magento.+?on.+?\n/gim, '');
+    contents.replace(/.+?Generated by @magento[\S\s]+?Z\.\n/gim, '');
```

Comment:
What's this part doing exactly?

Reply:
Removes the line generated by this string literal, which is dynamic and would cause the snap to fail.

```js
`Generated by @magento/pwa-buildpack v${buildpackVersion} on ${new Date().toISOString()}.`
```

The extra characters in `5.1.0-alpha.1` caused the string to break to a new line, and broke the regex pattern match.